integration-tests/utils/integration-utils.js

```diff
@@ -159,8 +159,17 @@ const select = async (page, labelContains, text) => {
   return click(page, `//div[starts-with(@id, "react-select-") and contains(normalize-space(.),"${text}")]`)
 }
 
-const waitForNoSpinners = page => {
-  return page.waitForXPath('//*[@data-icon="loadingSpinner"]', { hidden: true })
+// Used to prevent checking loadingSpinner while page is blank
+const waitUntilDomContentLoaded = page => {
+  return Promise.all([
+    page.waitForFunction(() => document),
+    page.waitForSelector('div:not(:empty):not([id="root"])')
```

Comment:
I still don't understand this function. This selector matches the initial "Loading Terra..." message as well as anything rendered after that message goes away. So why is this complex selector needed? It seems like we could just wait for the root div with `#root"`.

Reply:
I have removed `waitUntilDomContentLoaded`.